Compatibility issue with asynch==0.3.0

**Describe the bug**
Cannot open a connection to ClickHouse with `asynch==0.3.0` installed:
```
self = <clickhouse_sqlalchemy.drivers.asynch.connector.AsyncAdapt_asynch_dbapi object at 0x73b49b457680>
args = ('clickhouse://test:test@127.0.0.1:9000/test',), kwargs = {}

    def connect(self, *args, **kwargs) -> 'AsyncAdapt_asynch_connection':
        return AsyncAdapt_asynch_connection(
            self,
>           await_only(self.asynch.connect(*args, **kwargs))
        )
E       AttributeError: module 'asynch' has no attribute 'connect'. Did you mean: 'connection'?

.../clickhouse_sqlalchemy/drivers/asynch/connector.py:149: AttributeError
```

**To Reproduce**
```py
from asyncio import run
from sqlalchemy.ext.asyncio import create_async_engine


async def main() -> None:
    engine = create_async_engine('clickhouse+asynch://test:test@127.0.0.1:9000/test')

    connection = await engine.connect()

run(main())
```

**Expected behavior**
Connection is successful

**Versions**
`asynch==0.3.0`
`clickhouse-sqlalchemy==0.3.2`
`Python 3.12.8`

--- a/clickhouse_sqlalchemy/drivers/asynch/connector.py
+++ b/clickhouse_sqlalchemy/drivers/asynch/connector.py
@@ -146,7 +146,7 @@ class AsyncAdapt_asynch_dbapi:
     def connect(self, *args, **kwargs) -> 'AsyncAdapt_asynch_connection':
         return AsyncAdapt_asynch_connection(
             self,
-            await_only(self.asynch.connect(*args, **kwargs))
+            self.asynch.connection.Connection(*args, **kwargs)
         )
 
 

--- a/setup.py
+++ b/setup.py
@@ -99,7 +99,7 @@ setup(
         'sqlalchemy>=2.0.0,<2.1.0',
         'requests',
         'clickhouse-driver>=0.1.2',
-        'asynch>=0.2.2,<=0.2.4',
+        'asynch>=0.2.5',
     ],
     # Registering `clickhouse` as dialect.
     entry_points={

